composer.json

```
@@ -5,7 +5,7 @@
     "homepage": "http://drupal.org/project/bootstrap_sass",
     "license": "GPL-2.0+",
     "require": {
-        "drupal/bootstrap_barrio": "5.x-dev",
+        "drupal/bootstrap_barrio": "5.5.x-dev",
         "drupal/bootstrap_sass": "5.x-dev"
```

Thinking it might be a better idea to go slightly further and to use a wider spec here... if we just care about major version 5 here, then could do something like: 

```suggestion
        "drupal/bootstrap_barrio": "^5",
        "drupal/bootstrap_sass": "^5"
```

... then it's  up to whatever project is pulling this in to point at the dev branches as necessary/desirable.